app/lib/actions/getPharmacy.js

```diff
@@ -0,0 +1,18 @@
+const apiRequest = require('../etl-toolkit/apiRequest');
+const fixPhoneNumbers = require('../fixPhoneNumbers');
+const config = require('../config');
+
+function toObject(pharmaString, odsCode) {
+  const pharmacy = JSON.parse(pharmaString);
+  // eslint-disable-next-line no-underscore-dangle
+  pharmacy._id = odsCode;
```

Comment:
The _id field serves two purposes, an ID field is required by the etlStore to identify records that are added, and it is required in the output JSON (see comments below about why it is done here).

The etlStore will become part of a generic ETL toolkit. Rather than have the complication of setting the ID key to be used by the etlStore,  '_id' is used instead. This could be anything, but I've chosen it to be _id as this is useful as the key in mongoDB. If/when it becomes a full toolkit the _id field could always be removed after the processing is complete